src/common/args.h

```diff
@@ -64,6 +64,8 @@ enum class OptionsCategory {
     COMMANDS,
     REGISTER_COMMANDS,
 
+    SUBCOMMAND, // Specific to one or more subcommands
```

Comment:
In 50368d92ff0f878af2d6a70773bdf6a6e29b25bb "ArgsManager: support subcommand specific options"

I found this name to be confusing since the word "subcommand" implies that the option itself is a subcommand (i.e. the same category as `COMMANDS`), rather than it being an option to a command. Would suggest renaming this to something like `COMMAND_OPT`.

Reply:
Renamed to `COMMAND_OPTIONS` and changed "subcommand" to "command" more generally.